topic_monitor/topic_monitor/scripts/topic_monitor.py

```diff
@@ -24,6 +25,7 @@
 from std_msgs.msg import Float32, Header
 
 QOS_DEPTH = 10
+logger = rclpy.logging.get_named_logger('topic_monitor')
```

Comment:
This line fails for me with:

```
AttributeError: module 'rclpy' has no attribute 'logging'
```

Reply:
sounds like another side effect of https://github.com/ros2/rclpy/pull/147, will fix

Reply:
Well, the referenced PR was merged way before this one. So this change hasn't worked when is was merged.

Reply:
not trying to play the blame game here, just to understand and document what went wrong. The reason why that PR is relevant is because it broke many demos without us realising (and this PR was tested before that one was opened), and we intend to [investigate how to undo the side effects](https://github.com/ros2/rclpy/issues/156). therefore knowing that this is one of them is important

Reply:
https://github.com/ros2/demos/pull/210